layouts/partials/experience-description.html

```diff
@@ -4,6 +4,7 @@ <h2>{{ i18n "experience_title" }}</h2>
     <p class="lead">{{ i18n "experience_description" }}</p>
   </div>
   <div class="row">
+    {{ if .Site.Data.homepage.experience.button.icon }}
```

Comment:
Do you think it could make sense to still allow to display the section without an icon?

Maybe we could add some placeholder to indicate that the section content is wanted but the icon should not be present.

Reply:
What about the fix that i provided in 2b62591e39aa0c84887ea400b36485b0ba88d531 + c637c9b74cc7f72f2d2ba47aa6252eaf1162c2df

If the icon is not provided we hide the `<i>` element, but not the whole block - as there can still be text and URL provided (with no icon)

Reply:
Maybe we can add a new parameter, `enable` (true/false) to the whole button. 
What do you think @fhinok ?

Reply:
65ab4aff902fceec14f564938dfc4c8e17a2e703 looks like what I'd propose for this case :)